apps/emqx_bridge_syskeeper/src/emqx_bridge_syskeeper.erl

```diff
@@ -0,0 +1,133 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2022 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%--------------------------------------------------------------------
+-module(emqx_bridge_syskeeper).
+
+-include_lib("typerefl/include/types.hrl").
+-include_lib("hocon/include/hoconsc.hrl").
+-include_lib("emqx_bridge/include/emqx_bridge.hrl").
+-include_lib("emqx_resource/include/emqx_resource.hrl").
+
+-import(hoconsc, [mk/2, enum/1, ref/2]).
+
+-export([
+    bridge_v2_examples/1,
+    values/1
+]).
+
+-export([
+    namespace/0,
+    roots/0,
+    fields/1,
+    desc/1
+]).
+
+%% -------------------------------------------------------------------------------------------------
+%% api
+bridge_v2_examples(Method) ->
+    [
+        #{
+            <<"syskeeper_forwarder">> => #{
+                summary => <<"Syskeeper Forwarder Bridge">>,
+                value => values(Method)
+            }
+        }
+    ].
+
+values(get) ->
+    maps:merge(
+        #{
+            status => <<"connected">>,
+            node_status => [
+                #{
+                    node => <<"emqx@localhost">>,
+                    status => <<"connected">>
+                }
+            ]
+        },
+        values(post)
+    );
+values(post) ->
+    maps:merge(
+        #{
+            name => <<"syskeeper_forwarder">>,
+            type => <<"syskeeper_forwarder">>
+        },
+        values(put)
+    );
+values(put) ->
+    #{
+        enable => true,
+        connector => <<"syskeeper_forwarder">>,
+        target_topic => <<"${topic}">>,
+        target_qos => <<"-1">>,
+        template => <<"${payload}">>,
+        resource_opts => #{
+            worker_pool_size => 16
+        }
+    }.
+
+%% -------------------------------------------------------------------------------------------------
+%% Hocon Schema Definitions
+namespace() -> "bridge_syskeeper".
+
+roots() -> [].
+
+fields(config) ->
+    [
+        {enable, mk(boolean(), #{desc => ?DESC("config_enable"), default => true})},
+        {description, emqx_schema:description_schema()},
+        {connector,
+            mk(binary(), #{
+                desc => ?DESC(emqx_connector_schema, "connector_field"), required => true
+            })},
+        {target_topic,
+            mk(
+                binary(),
+                #{desc => ?DESC("target_topic"), default => <<"${topic}">>}
+            )},
+        {target_qos,
+            mk(
+                range(-1, 2),
+                #{desc => ?DESC("target_qos"), default => -1}
+            )},
+        {template,
+            mk(
+                binary(),
+                #{desc => ?DESC("template"), default => <<"${payload}">>}
+            )},
```

Comment:
we need to move these down to a struct of type named `parameters` 
wonder why this test did not fail for you:
https://github.com/emqx/emqx/blob/053c03f79f54d1667f9d045efca497ef61e92d92/apps/emqx_bridge/src/schema/emqx_bridge_v2_schema.erl#L166-L170